lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1743,6 +1743,8 @@ sub store_column {
         }
         # make sure no modules are left running
         $self->modules->search({result => RUNNING})->update({result => NONE});
+        my $app = OpenQA::App->singleton;
+        $app->gru->enqueue(finalize_job_results => [$self->id]) if $app;
```

Comment:
When is this false?

Reply:
*Maybe* in tests.

Reply:
Yes, test fixtures cannot be created if I enqueue the minion task unconditionally.